src/api/language.js

```diff
@@ -24,6 +30,21 @@ const loadMessages = (lang) => {
     .then((messages) => Object.assign.apply({}, messages))
 }
 
+const loadAllLanguagesMessages = () => {
```

Comment:
We could get rid off of this `loadAllLanguagesMessages` altogether, not sure if it's a handy helper to keep around

Reply:
should we remove this? I don't see how we can use this now and future.

Reply:
Yea I think we should, will push a commit for this